com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/font/Fontc.java

```diff
@@ -765,6 +739,10 @@ else if (channelCount==3)
                 throw new FontFormatException("Could not generate font preview: " + e.getMessage());
             }
         }
+
+        boolean monospace = true;
```

Comment:
Maybe we could start with e.g. `boolean monospace = include_glyph_count > 1`? Then we wouldn't need `if (include_glyph_count <= 1) monospace = false` at the end.